activerecord/lib/active_record/associations/join_dependency.rb

```diff
@@ -228,8 +228,8 @@ def walk(left, right)
         end
 
         def find_reflection(klass, name)
-          klass._reflect_on_association(name) or
-            raise ConfigurationError, "Can't join '#{ klass.name }' to association named '#{ name }'; perhaps you misspelled it?"
+          klass._reflect_on_association(name) ||
+            raise( ConfigurationError, "Can't join '#{ klass.name }' to association named '#{ name }'; perhaps you misspelled it?")
```

Comment:
Extra space before the open parenthesis. Shouldn't we also remove the spaces within the interpolations?